Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -12,22 +12,24 @@ import SwiftUI
 
 struct MenuItemHeaderView: View {
     @Environment(\.colorScheme) private var colorScheme: ColorScheme
-    var icon: UIImage
+    var icon: UIImage?
```

Comment:
I don't think this has to be optional? in code below you fallback to an empty UIImage regardless